vbezbakh/docker/vbezbakh:devopscampsummer2019_vbezbakh_docker_example3/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu
+RUN apt-get update
+RUN apt-get -y install sl
+RUN ln -s /usr/games/sl /usr/bin/sl
+ENTRYPOINT ["sl", "-l"]
```

Comment:
Entrypoint command arguments would be good to move into the CMD option if it isn't necessary.